File: SwiftPMPackages/SwiftPackage/Sources/SwiftPackage/SwiftPackage.m
//
//  SwiftPackage.m
//  SwiftPackage
//
//  Created by Resit Berkay Bozkurt on 02.06.25.
//

#import "SwiftPackage.h"

@implementation SwiftPackage

- (instancetype)init {
    self = [super init];
    if (self) {
        _desc = @"This is SPM package code";
    }
    return self;
}

- (NSString *)describe {
    return self.desc;
}

@end

File: SwiftPMPackages/SwiftPackage/Sources/SwiftPackageSwift/SwiftPackage.swift
//
// Created by Resit Berkay Bozkurt on 03.06.25.
//
import Foundation

@objc public class SwiftPackage: NSObject {

    @objc public var desc: String

    @objc public override init() {
        self.desc = "This is SPM package Swift code"
        super.init()
    }

    @objc public func describe() -> String {
        return desc
    }
}
